src/db/models/article.py

```diff
@@ -14,3 +15,10 @@ class Article(Base):
     link = Column(String())
```

Comment:
Jak już mamy tabele na linki, to nie trzeba odzielnego pola na link w modelu `Article`  trzymać, można zdefiniować relacje do tablicy z linkami.
```python
article_link= relationship("Links", uselist=False, backref="pilkanoznapl")
```
`uselist` definiuje relacje 1:1